src/test/java/org/mockbukkit/mockbukkit/inventory/InventoryViewMockTest.java

```diff
@@ -90,18 +98,129 @@ void getType_TypeSet_SameReturned()
 		assertEquals(InventoryType.CREATIVE, view.getType());
 	}
 
-    @Test
-    void getOriginalTitle()
+	@Test
+	void getOriginalTitle()
 	{
 		view.setTitle("Test");
 		view.setTitle("Foobar");
 		assertEquals("Inventory", view.getOriginalTitle());
-    }
+	}
 
-    @Test
-    void setTitle()
+	@Test
+	void setTitle()
 	{
 		view.setTitle("Test");
 		assertEquals("Test", view.getTitle());
-    }
+	}
+
+	@Test
+	void getItemFromTopInventory()
+	{
+		ItemStack sword = ItemStack.of(Material.IRON_SWORD);
+		Player player = server.addPlayer();
+		InventoryMock chest = new ChestInventoryMock(null, 9);
+		chest.setItem(0, sword);
+		view = new PlayerInventoryViewMock(player, chest);
+
+		assertEquals(sword, view.getItem(0));
+	}
+
+	@Test
+	void getItemWrongIndex_1()
+	{
+		Player player = server.addPlayer();
+		InventoryMock chest = new ChestInventoryMock(null, 9);
+		view = new PlayerInventoryViewMock(player, chest);
+
+		assertNull(view.getItem(-1));
+	}
+
+	@Test
+	void getItemWrongIndex100()
+	{
+		Player player = server.addPlayer();
+		InventoryMock chest = new ChestInventoryMock(null, 9);
+		view = new PlayerInventoryViewMock(player, chest);
+
+		assertThrows(IndexOutOfBoundsException.class, () -> view.getItem(100));
+	}
+
+	@Test
+	void getItemFromBottomInventory()
+	{
+		ItemStack sword = ItemStack.of(Material.IRON_SWORD);
+		Player player = server.addPlayer();
+		player.getInventory().setItem(0, sword);
+		InventoryMock chest = new ChestInventoryMock(null, 9);
+		view = new PlayerInventoryViewMock(player, chest);
+
+		assertEquals(sword, view.getItem(9));
+	}
+
+	@Test
+	void setItemInTopInventory()
+	{
+		ItemStack sword = ItemStack.of(Material.IRON_SWORD);
+		Player player = server.addPlayer();
+		InventoryMock chest = new ChestInventoryMock(null, 9);
+		view = new PlayerInventoryViewMock(player, chest);
+		view.setItem(0, sword);
+
+		assertEquals(sword, chest.getItem(0));
+	}
+
+	@Test
+	void setItemInBottomInventory()
+	{
+		ItemStack sword = ItemStack.of(Material.IRON_SWORD);
+		Player player = server.addPlayer();
+		InventoryMock chest = new ChestInventoryMock(null, 9);
+		view = new PlayerInventoryViewMock(player, chest);
+		view.setItem(9, sword);
+
+		assertEquals(sword, player.getInventory().getItem(0));
+	}
+
+	@Test
+	void setItemNegativeSlot_WithNullItem()
+	{
+		Player player = server.addPlayer();
+		InventoryMock chest = new ChestInventoryMock(null, 9);
+		view = new PlayerInventoryViewMock(player, chest);
+
+		assertThrows(
```

Comment:
No index out of bounds exception is thrown when the item is null on a negative index. Nothing happens when this is done